recipes/libproperties/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, CMake, tools
+
+
+class LibpropertiesConan(ConanFile):
+    name = "libproperties"
+    version = "1.1.2"
+    license = "Apache-2.0 License"
+    author = "libbylg@126.com"
+    url = "https://github.com/tinyhubs/libproperties"
+    description = "libproperties is a library to parse the Java .properties files. It was writen in pure C. And fullly compatible with the Java .properties file format."
+    topics = (".properties", "Java", "pure C")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
```

Comment:
If you use my cmake script suggestion, then you can add `fPIC` here.
```suggestion
    options = {"shared": [True, False], "fPIC": [True, False],}
    default_options = {"shared": False, "fPIC": True,}
```

Reply:
Should I add the `-fPIC` option  to the CMakeLists.txt?

Reply:
No, conan can handle this by defining [`CMAKE_POSITION_INDEPENDENT_CODE`](https://cmake.org/cmake/help/latest/variable/CMAKE_POSITION_INDEPENDENT_CODE.html)